### DISCUSSION TITLE
是否可以提供电商相关的插件实现商品订单支付等功能类似wordpress 一样？

### DISCUSSION THREAD

**Discussion by RooneyLway** [halo-dev/halo#7036] [Ideas]
### 前置条件

- [x] 已经在 [Issues](https://github.com/halo-dev/halo/issues) 列表中搜索了相关问题。
- [x] 这是和 Halo 相关的特性，如果是非项目本身的问题，建议在 [Discussions](https://github.com/halo-dev/halo/discussions) 提交。
- [x] 如果是插件和主题特性建议，请在对应的插件和主题仓库提交。

### 描述一下此特性

急需上架商品功能，商品详情页面，商品价格，联系客服等。

目前有很多用户有这个需求，去做独立站，仅仅是企业建站官网的市场份额不大，这是很大的市场份额建议调研重视。

做产品要利他，不然用户怎么会选择这个产品，用户需要盈利。建议至少做一个商品详情页的插件，支付交易不是那么重要，跨境初期可以引流到亚马逊付款。

### 附加信息

急需上架商品功能

**Comment by ruibaby** [comment]
感谢反馈，我们有这方面的计划，并且我们自己运营的网站已经使用 Halo 实现商务的功能了，这也是为开发通用插件做积累。

https://www.halo.run/showcase/websites?tag=business-website

/kind feature
/area plugin
